src/website/src/pages/ec/help/faq/index.md

```diff
@@ -0,0 +1,143 @@
+- [@ecl/ec-preset-website](https://www.npmjs.com/package/@ecl/ec-preset-website)
+- [@ecl/ec-preset-full](https://www.npmjs.com/package/@ecl/ec-preset-full)
+- [@ecl/ec-preset-legacy](https://www.npmjs.com/package/@ecl/ec-preset-legacy)
+- [@ecl/ec-preset-legacy-website](https://www.npmjs.com/package/@ecl/ec-preset-legacy-website)
+
+To access EU system presets, please use the same links as above and change `ec-` to `eu-`.
+
+When trying to decide which preset is the most appropriate for your project needs, have in mind the following:
+
+- `preset-website` is the most popular start. This preset ships all components with normalize.css and some styles applied to the document body.
+- `preset-full` is the same as `preset-website` but does not contain normalise.css and does not apply any global styles. If you are not starting the ECL implementation from scratch, but on an existing website, then this could be more appropriate preset.
+- `preset-editor` provides styles for paragraphs, lists and other generic content-related HTML elements which cannot have the necessary ECL classes applied programatically. Most frequently used in CMS or frameworks which manage content through WYSIWYG editors rather than view templates.
+
+In terms of how ECL-based websites follow up with the library's evolution when components get deprecated:
+
+- `preset-legacy-website` is an easy upgrade path to websites which have started off `preset-website` and have experienced less frequent upgrades. If any component used in `preset-website` has been deprecated and has become unavailable to a higher version, the deprecated will remain available in `preset-website-legacy` for backwards compatibility.
+- `preset-legacy` contains what has been deprecated in time.
+
+Using presets is a highly-recommended approach of implementing ECL.
+
+## Can I build/use a customised version of ECL?
+
+Short answer: yes, you can.
+
+Long answer: probably you should not. The main reason of using ECL in first place is to get all the benefits of ready-made CSS and JavaScript rules which comply with strict web governance rules.
+
+Yet, there are cases when you need to both get the compliance, but also make small tweaks or overrides.
+
+If this is the case, the heaviest approach would be to clone the above-mentioned Github repository and build the project manually. The process is heavy in terms of resource needs (the project has big history) and time to grasp project organization's sets of tools and dependencies.
+
+There's a smaller step you can make in the process of building customized version of ECL. You can make use of [`@ecl/builder`](https://www.npmjs.com/package/@ecl/builder) CLI utility which can facilitate your bundling and optimisation tasks. With `@ecl/builder` many choices have already been made for you: choice of bundler, optimisation strategies, etc.
+
+Please note that there is no support for customized versions of ECL and the practice is not recommended.
```

Comment:
In my opinion this warning should also be on the first page documentation. But i also get that from maybe a business point of view we keep this here in the faq